routers/users.py

```
@@ -0,0 +1,369 @@
+        try:
+            contents = await profileImage.read()
+            img = Image.open(io.BytesIO(contents))
+
+            if img.format.lower() not in ['jpeg', 'png']:
+                validation_errors.append({"field": "profileImage", "issue": "유효하지 않은 이미지 포맷입니다."})
+
+            if img.width < 500 or img.height < 500:
+                validation_errors.append({"field": "profileImage", "issue": "사이즈는 500x500px 이상만 가능합니다."})
+
+            await profileImage.seek(0)  # 파일 포인터 초기화
+        except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)